systemd/usr/libexec/miriway-session-startup

```diff
@@ -55,6 +55,7 @@
 #  - https://www.freedesktop.org/software/systemd/man/systemd.special.html#graphical-session.target
 #  - https://systemd.io/DESKTOP_ENVIRONMENTS/
 #
+export XDG_CURRENT_DESKTOP=mir:Miriway
```

Comment:
Why change this? The default configuration has this as "Miriway", if another desktop configuration changed it, should it be overridden?